src/classes/ADDR_AdminAcc_TEST.cls

```diff
@@ -38,338 +38,356 @@
 @isTest
 public with sharing class ADDR_AdminAcc_TEST {
     
+    /*@description Set up of Hierarchy Settings leverage by test methods*/
     private static void configSettings() {
-        Hierarchy_Settings__c hs = new Hierarchy_Settings__c(
-            Accounts_Addresses_Enabled__c = UTIL_Describe_API.getAdminAccRecTypeID() + ';',
-            Simple_Address_Change_Treated_as_Update__c = true);
+        Hierarchy_Settings__c hs = new Hierarchy_Settings__c(Accounts_Addresses_Enabled__c = UTIL_Describe_API.getAdminAccRecTypeID() + ';',
```

Comment:
Maybe not related to this item...Should we use UTIL_CustomSettingsFacade.getSettingsForTests?